src/components/containers/ProfileMain/ProfileMain.jsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/display-name */
+import React, { Component } from 'react';
+import { Tab, Icon } from 'semantic-ui-react';
+
+import Title from '../../presentationals/Title/Title.jsx';
+import ArticleLists from '../ArticleLists/ArticleLists.jsx';
+import Settings from '../ProfileSettings/ProfileSettings.jsx';
+import './ProfileMain.scss';
+
+class ProfileMain extends Component {
```

Comment:
We can change this to stateless component.

Reply:
I have changed this.